sysutils/pfSense-pkg-apcupsd/Makefile

```diff
@@ -27,6 +27,8 @@ do-extract:
 do-install:
 	${MKDIR} ${STAGEDIR}${PREFIX}/pkg
 	${MKDIR} ${STAGEDIR}${PREFIX}/www
```

Comment:
This line can should be removed since following line will also create this directory.  ${MKDIR} uses -p parameter

Reply:
Removed